test/evergreen.yml

```diff
@@ -364,6 +364,9 @@ functions:
           echo "$exit_code" > cppsuite_exit_code
           if [ "$exit_code" != 0 ]; then
             echo "[{\"info\":{\"test_name\": \"${test_name}\"},\"metrics\": []}]" > ${test_name}.json
+            # Since we later remove the WiredTiger folder, we need to check for core dumps now.
+            cd ../..
+            ${python_binary|python3} ../test/evergreen/print_stack_trace.py -c test/cppsuite -l .
```

Comment:
Just a curious question why do we remove the WiredTiger folder later here?

Reply:
This was done as part of WT-9479 by @luke-pearson. In this set of tasks, we explicitly upload the cppsuite folder but this is done too during the post tasks. In order to avoid doing it twice, we upload it here, delete the folder and the post task has nothing to upload.